packages/manager/modules/pci/src/projects/project/billing/billing.routing.js

```diff
@@ -6,6 +6,19 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/billing',
     controller,
     controllerAs: 'BillingCtrl',
+    redirectTo: (transition) => {
```

Comment:
You can't inject `$q` here ?

Reply:
The `redirectTo` function is passed just 3 parameters, which are fixed (as described here: https://docs.angularjs.org/api/ngRoute/provider/$routeProvider)

Hence, any dependencies has to be got from the injector :)